charts/dz-control-plane-deps/Makefile

```diff
@@ -146,8 +146,9 @@ install-devzero-db:
 		REPO_URL=https://charts.bitnami.com/bitnami \
 		REPO_APP_NAME=postgresql \
 		VERSION=16.3.5 \
-		NAMESPACE=devzero
-
+		NAMESPACE=devzero \
+		EXTRA_ARGS='--set primary.resources.requests.memory=512Mi \
```

Comment:
This should have been fixed with this https://github.com/devzero-inc/self-hosted/pull/111